content/projects/data-science-specific/k-means-clustering/_index.md

```diff
@@ -28,14 +28,22 @@ title: K-Means Clustering Assignment
 
 ## Pre-requisites
 
-Read through the K-Means Tutorials at {{< contentlink path="topics/data-science-specific/clustering" >}} before starting this project.
+Read through the K-Means Tutorials at [content link](http://syllabus.africacode.net/topics/data-science-specific/clustering/) before starting this project.
 
-## Clustering whisky distilleries according to tasting Profiles
+## About the DataSet
```

Comment:
Why is dataset capitalised like that? Is it a class?

Reply:
This isn't a class; we write "DataSet" like this to emphasize the importance of the dataset. Alternatively, we can represent it as "Dataset."

Reply:
Agree "DataSet" would be better as "dataset" or if need be as "Dataset" - but good that we try have consistency across projects in how we use capitalisation in the headings